packages/client/components/NewGitHubIssueInput.tsx

```diff
@@ -79,30 +84,49 @@ const SearchInput = styled('input')({
   width: '100%'
 })
 
+const Error = styled(StyledError)({
+  fontSize: 13,
+  textAlign: 'left',
+  width: '100%'
+})
+
 interface Props {
   isEditing: boolean
   meeting: NewGitHubIssueInput_meeting
   setIsEditing: (isEditing: boolean) => void
   viewer: NewGitHubIssueInput_viewer
 }
 
+const validateIssue = (issue: string) => {
+  return new Legitity(issue)
+    .trim()
+    .min(2, 'C’mon, you call that an issue?')
+    .max(255, "Who's gonna read all that? Short and sweet please.")
```

Comment:
:rofl: nice
you can always use a smart quote `’` to avoid using an escape (plus it gives it a nice polished look)

Reply:
Nice, added!